mix.exs

```diff
@@ -8,7 +8,15 @@ defmodule GoldenPath.MixProject do
       elixir: "~> 1.5",
       elixirc_paths: elixirc_paths(Mix.env()),
       compilers: [:phoenix, :gettext] ++ Mix.compilers(),
+      dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:mix]],
       start_permanent: Mix.env() == :prod,
+      test_coverage: [tool: ExCoveralls],
```

Comment:
Elixir has really strong utilities around ExUnit output, but this makes it really, really simple to configure the kind of output you like either locally or to integrate with a CI Server of your choice!